.github/workflows/ci_test_build_package.yml

```diff
@@ -0,0 +1,21 @@
+# # This workflow runs all available tests and builds the .jar files
+name: Varianstore CI
```

Comment:
Note: A descriptive name what the workflow actually does. This name will pop up in the workflow overview once it runs.